kcalign/__init__.py

```diff
@@ -561,7 +563,10 @@ def check_tab(tab):
         if int(tab) in [1, 2, 3, 4, 5, 6, 9, 10, 11, 12, 13, 14, 15, 16, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 33]:
```

Comment:
use a range here!

Reply:
and do the `int` conversion outside of the function to avoid the duplicated action